getting-started/kubernetes/vpp/eks.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Install VPP on an EKS cluster
+description: Install Calico with the VPP dataplane on an EKS cluster
+canonical_url: '/getting-started/kubernetes/vpp/eks'
+---
+
+### Requirements
```

Comment:
### Requirements

* The AWS CLI installed on your system
* A version of `kubectl` installed, that supports your EKS cluster
* An EKS cluster configured (without Fargate)